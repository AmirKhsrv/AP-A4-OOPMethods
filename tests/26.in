show_author_info 1529
best_reviewer
best_reviewer
