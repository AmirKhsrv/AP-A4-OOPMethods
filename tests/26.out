id: 1529
Name: Báº£o Ninh
Year of Birth: 1985
Place of Birth: Argentina
Member Since: October 22 2019
Genres: Science Fiction
Books:
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
